types/nanoid/non-secure/generate.d.ts

```diff
@@ -0,0 +1,3 @@
+declare function generate(alphabet: string, size?: number): string;
```

Comment:
The underlying library appears to have nice doc comments for these functions.  Consider copying them over so that they appear in IntelliSense.